### PR TITLE
feat(spacectl/stack): add stack output command

### PR DESCRIPTION
**Pull request by garry-harthill-cko**
Add a command to the stack menu which displays the current outputs
of the stack.

### CODE REVIEW

**Comment by jmfontaine** [comment]
Thanks for the PR @garry-harthill-cko! I will have the Engineering team review it.

File: internal/cmd/stack/stack.go [hunk 1]
```diff
@@ -174,6 +174,19 @@ func Command() *cli.Command {
 					},
 				},
 			},
+			{
+				Name:  "outputs",
+				Usage: "Shows current outputs for a specific stack. Does not show sensitive outputs.",
```
Comment (adamconnelly) [inline internal/cmd/stack/stack.go:179]: Maybe we need to tweak this to "Does not show the value of sensitive outputs", on the grounds you're displaying them but just masking the value?

**Review by adamconnelly — COMMENTED** [review]
@garry-harthill-cko thanks a lot for the contribution - it looks great! I've just left a small suggestion about the usage message for the command. Let me know what you think.

**Comment by garry-harthill-cko** [comment]
> @garry-harthill-cko thanks a lot for the contribution - it looks great! I've just left a small suggestion about the usage message for the command. Let me know what you think.

Good point. It does still display the sensitive outputs. Just not the values.

**Review by adamconnelly — APPROVED** [review]
LGTM - thanks again Garry :bow:

**Pull request merged**